.eslintrc.js

```diff
@@ -54,37 +54,50 @@ module.exports = {
 		'lines-between-class-members': [1],
 		'max-len': [0],
 		'newline-per-chained-call': [0, { 'ignoreChainWithDepth': 2 }],
-		'no-mixed-operators': [0],
+		'no-mixed-operators': [1],
```

Comment:
Same, remove this line.

Reply:
updated in feature/cleanup-cleanup as this PR was already merged.

Reply:
lgtm in feature/cleanup-cleanup